app/tests/unit/lib/theme/widgets/AdjustableCalculatedValueWidget.test.tsx

```diff
@@ -70,4 +70,23 @@ describe("The AdjustableCalculatedValueWidget", () => {
 
     expect(props.onChange).toHaveBeenLastCalledWith(undefined);
   });
+
+  it("renders the help tooltip", () => {
+    const props: any = {
+      uiSchema: { isMoney: true, calculatedValueFormContextProperty: "myProp" },
+      schema: { type: "number", title: "Test Label", default: undefined },
+      id: "test-id",
+      label: "Test Label",
+      onChange: jest.fn(),
+      value: "1",
+      formContext: { myProp: "100" },
+    };
+    render(<AdjustableCalculatedValueWidget {...props} />);
+
+    const helpIcon = screen.getByLabelText("test-label-(adjusted)-tooltip");
+    expect(helpIcon).toBeInTheDocument();
+
+    fireEvent.mouseOver(helpIcon);
+    expect(screen.getByRole("tooltip")).toBeInTheDocument();
```

Comment:
Can we check for the text of the tooltip here?

Reply:
done!